Use x-www-browser by default.
--- a/src/clfswm-util.lisp
+++ b/src/clfswm-util.lisp
@@ -735,7 +735,7 @@
 ;;;;;| Various definitions
 ;;;;;`-----
 
-(defun show-help (&optional (browser "dillo") (tempfile "/tmp/clfswm.html"))
+(defun show-help (&optional (browser "x-www-browser") (tempfile "/tmp/clfswm.html"))
   "Show current keys and buttons bindings"
   (ignore-errors
     (produce-doc-html-in-file tempfile))
